submit.go

```diff
@@ -303,40 +353,45 @@ func (s *submitServer) parseRequest(w http.ResponseWriter, req *http.Request, re
 		p, err = parseMultipartRequest(w, req, reportDir)
 		if err != nil {
 			log.Println("Error parsing multipart data:", err)
-			http.Error(w, "Bad multipart data", 400)
+			http.Error(w, "Bad multipart data", http.StatusBadRequest)
 			return nil
 		}
 	} else {
 		p, err = parseJSONRequest(w, req, reportDir)
 		if err != nil {
 			log.Println("Error parsing JSON body", err)
-			http.Error(w, fmt.Sprintf("Could not decode payload: %s", err.Error()), 400)
+			http.Error(w, fmt.Sprintf("Could not decode payload: %s", err.Error()), http.StatusBadRequest)
 			return nil
 		}
 	}
 
-	if p.AppName == "booper" {
-		if gplaySpamEmailRegex.MatchString(p.Data["user_id"]) {
-			log.Println("Dropping report from", p.Data["user_id"])
```

Comment:
I think we still need this filter

Reply:
Why? It seems like a bad heuristic. What if someone just happens to have a bunch of numbers at the end of their gmail account?

Reply:
We don't want to be spammed by the google play testers in linear, they're still sending over a dozen bug reports per day

Reply:
I used to have a gmail account with 7 numbers at the end.

I went ahead and added this check back in, but this might be something we want to revisit.